### ISSUE TITLE
Not Working In Production Mode

### ISSUE THREAD

**Issue by larryh** [winton/acts_as_archive#23]
Okay, I'm probably doing something stupid...

I'm using Ruby 1.9.2, Rails 3.0.3 and acts_as_archive 0.4. I already posted in the "Hi there" thread about the yml file not working, but once I put 'acts_as_archive' in my model it worked fine.

That's still the case - in development mode. When I did that my archive table was created. 

However, when I run db:migrate RAILS_ENV=production the archive table doesn't get created in my production database.  Any ideas?

Thanks,
Larry

**Comment by elmatou** [comment]
Since Winton does'nt response to any issue...
can you run the migration with --trace option and dump de output ?!
Maybe we will find smthing

**Comment by larryh** [comment]
Hi,

Thanks for responding. I have my environment duplicated on a laptop and when I tried creating the table in the production database on that machine it worked. 

The one thing I did differently was this: on my desktop in between creating the archive table in development and production modes I ran db:test:prepare. I did not do this on the laptop.

So I deleted the archive table on my desktop and ran db:migrate in development mode and then production mode and it worked fine. 

Don't know if the problem was due to running db:test:prepare or else can be attributed to one of those weird glitches we all experience (none of which are our fault ;-), but the problem is solved.

Thanks a lot for trying to help,
Larry
